src/amo/components/Addon/styles.scss

```diff
@@ -160,6 +162,7 @@
   }
 }
 
+.Addon .AMInstallButton-button,
```

Comment:
I am not sure about that, but it was set for `InstallButton` so I added it..

Reply:
ah this one fixes that:

Before

![screen shot 2018-08-28 at 17 53 51](https://user-images.githubusercontent.com/217628/44734963-6c8cbf80-aaeb-11e8-89ac-7c880596873e.png)


After

![screen shot 2018-08-28 at 17 53 46](https://user-images.githubusercontent.com/217628/44734952-6991cf00-aaeb-11e8-9451-f2cbba00a5f1.png)

Reply:
just checking...do we want this? on stage it is full width on ~ mobile (with the installButton styles). This (full width on mobile) is also consistent with the "..get firefox now" button on non-firefox (chrome) for example. Or should it be smaller now in this case?

Reply:
this is not mobile here

Reply:
ok cool, I see. 👍  then : )